src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <IncludeBuildOutput Condition="'$(TargetFramework)' == 'netstandard2.0'">false</IncludeBuildOutput>
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+    <PackageLicenseExpression></PackageLicenseExpression>
```

Comment:
This package is not MIT licence ?

Reply:
Nope. This package is actually code from Intel that we are compiling to remove the code we don't need (reduces the size by a LARGE amount. Like 90% or so). So its one of Intel's licenses and not MIT. Only package we have that is different like this.